### PR TITLE
misc: Process args for subdimensions

### PR DESCRIPTION
**Pull request by mloubout**

### CODE REVIEW

File: devito/types/dimension.py [hunk 1]
```diff
@@ -889,6 +889,23 @@ def free_symbols(self):
             pass
         return retval
 
+    def _arg_values(self, interval, grid=None, **kwargs):
+        # Parent dimension define the interval
+        fact = self._factor.data if self._factor is not None else 1
+        toint = lambda x: int(np.ceil(x/fact))
+        vals = dict()
```
Comment (FabioLuporini) [inline devito/types/dimension.py:896]: = {} 

:D

File: devito/types/dimension.py [hunk 2]
```diff
@@ -889,6 +889,23 @@ def free_symbols(self):
             pass
         return retval
 
+    def _arg_values(self, interval, grid=None, **kwargs):
+        # Parent dimension define the interval
+        fact = self._factor.data if self._factor is not None else 1
+        toint = lambda x: int(np.ceil(x/fact))
```
Comment (FabioLuporini) [inline devito/types/dimension.py:895]: probably just math.ceil is OK

**Review by FabioLuporini — APPROVED** [review]

File: devito/types/dimension.py (hunk 1 above)
Comment (mloubout) [reply devito/types/dimension.py]: fixed

File: devito/types/dimension.py (hunk 2 above)
Comment (mloubout) [reply devito/types/dimension.py]: ok done

**Pull request merged**